docs/getting-started/cad-software.md

```diff
@@ -6,34 +6,23 @@ Setup instructions on how install CAD software for schematic capture and PCB lay
 In the past, the team has used KiCAD. However, in the fall of 2017, we began to move all the schematics to Altium. 
 
 ## Altium Setup
+Altium is only compatible with Windows. We currently use **version 18.0**. For more information on the system requirements for this version of Altium, go [here](https://www.altium.com/documentation/altium-designer/altium-designer-system-requirements?version=18.0).
 
-{% hint style="warning" %}
-Altium is only compatible with Windows. We currently use version 16.0. For more information on the system requirements for this version of Altium, go [here](https://www.altium.com/documentation/16.0/display/ADES/Altium+Designer+-+%28%28System+Requirements%29%29).
-{% endhint %}
+### Part 1 - Installation
 
-### ECE Students
+1. Create an [Altium Live account](https://live.altium.com/#Join). Once the request is sent, it may take a few days for approval. Note: If you do not see any entry fields for your information on the linked page, try a different browser.
```

Comment:
Is the email address used when creating the account not important? I thought the email address had to granted license permission through ece. This is what I remember doing as an IGEN student.

Reply:
I used my personal email and it was fine. However, I will add that this should be an UBC alum email